src/Utility/Discord/ReactionCollector.js

```diff
@@ -0,0 +1,173 @@
+import { EventEmitter } from 'eventemitter3';

```

Comment:
We dropped eventemitter3 in 126d39b383dd438fd68d3ed5a5ec3a1a8b0cccbe.
```suggestion
import { EventEmitter } from 'events';
```